Test fail related to monotonic clock reading,

Hi

When running test of this lib, found following issue.

**What's the issue?**
Test fail with message:
```
--- FAIL: TestConversion (0.00s)
    conversion_test.go:110:
                Error Trace:    conversion_test.go:110
                Error:          Not equal:
                                expected: time.Time{wall:0xbf0d2129064e75f4, ext:4025401, loc:(*time.Location)(0xa33dc0)}
                                actual  : time.Time{wall:0x64e75f4, ext:63684609060, loc:(*time.Location)(0xa33dc0)}

                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1,2 +1,2 @@
                                -(time.Time) 2019-02-01 17:11:00.1058053 +0800 CST m=+0.004025401
                                +(time.Time) 2019-02-01 17:11:00.1058053 +0800 CST

                Test:           TestConversion
FAIL
exit status 1
```
**How to repro?**
Win10, run `go test` with latest code.

Possibly related to [monotonic clock reading](https://golang.org/pkg/time/#Time.String), formal time formatting in test might help to fix.

Best Regards,
Jiachen

--- a/conversion_test.go
+++ b/conversion_test.go
@@ -107,7 +107,7 @@ func TestConversion(t *testing.T) {
 	assert.NotEqual(t, uintptr(0), sys.TargetName)
 	assert.Equal(t, cred.TargetName, res.TargetName)
 	assert.Equal(t, cred.Comment, res.Comment)
-	assert.Equal(t, cred.LastWritten, res.LastWritten)
+	assert.True(t, cred.LastWritten.Equal(res.LastWritten))
 	assert.Equal(t, cred.TargetAlias, res.TargetAlias)
 	assert.Equal(t, cred.UserName, res.UserName)
 	cred.TargetName = "Another Foo"

